src/rust/wpcomics/sources/xoxocomics/res/source.json

```
@@ -3,7 +3,7 @@
 		"id": "en.xoxocomics",
 		"lang": "en",
 		"name": "XOXO Comics",
-		"version": 2,
+		"version": 3,
 		"url": "https://xoxocomic.com/",
```

shouldn't have a trailing slash
```suggestion
		"url": "https://xoxocomic.com",
```